hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java

```diff
@@ -216,7 +217,9 @@ public void notifyConfigurationChanged(long term, long index,
     List<RaftProtos.RaftPeerProto> newPeers =
         newRaftConfiguration.getPeersList();
     LOG.info("Received Configuration change notification from Ratis. New Peer" +
-        " list:\n{}", newPeers);
+        " list:\n[{}\n]", newPeers.stream().map(peer ->
+            String.format("id: \"%s\"%n address: \"%s\"", peer.getId().toStringUtf8(), peer.getAddress()))
+        .collect(Collectors.joining("\n, ")));
```

Comment:
I would prefer to format the string without newlines.  Log messages split into several lines are prone to be mixed up with messages from other threads.

Also, would be nice to move the processing out of the log statement.

```suggestion
    List<String> formatted = newPeers.stream()
        .map(peer -> String.format("id: \"%s\" address: \"%s\"", peer.getId().toStringUtf8(), peer.getAddress()))
        .collect(Collectors.toList());
    LOG.info("Received Configuration change notification from Ratis. New Peer list: {}", formatted);
```

Reply:
I originally intended to maintain the original log style. Thanks for reviewing!

After the modification, the current log：
```=bash
Received Configuration change notification from Ratis. New Peer list: [id: "omNode-3" address: "localhost:15015", id: "omNode-1" address: "localhost:15007", id: "omNode-2" address: "localhost:15011"]
```

Reply:
Thanks @harryteng9527 for updating the patch.